src/main/java/com/fauna/query/StreamTokenResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.fauna.query;
+
+import com.fauna.annotation.FaunaField;
+
+public class StreamTokenResponse {
+    @FaunaField(name = "@stream")
+    private String token;
```

Comment:
This doesn't seem right. If we're getting a tagged type back, that should be handled in the UTF8FaunaParser and be converted into a new token type, like `FaunaTokenType.STREAM`

Reply:
Hmm, OK.